src/closures/constant_diffusivity_closures.jl

```diff
@@ -0,0 +1,83 @@
+#
+# Constant diffusivity
+#
+
+"""
+    ConstantIsotropicDiffusivity(T=Float64; ν=1e-6, κ=1e-7)
+
+Return a `ConstantIsotropicDiffusivity` closure object of type `T` with
+viscosity `ν` and scalar diffusivity `κ`.
+"""
+Base.@kwdef struct ConstantIsotropicDiffusivity{T} <: ScalarDiffusivity{T}
+    ν :: T = 1e-6
```

Comment:
These seem like arbitrary defaults. Why not use the actual molecular values for water?

Can we print a warning if a user uses these defaults? Would that make sense? I imagine most people would actually want to use some sort of eddy diffusivity?